src/components/Test/Quote.test.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import Quote from '../Quote';
+
+describe('Quote component', () => {
+  it('should render quote and author correctly', () => {
+    const quoteText = 'This is a test quote';
+    const authorName = 'Test Author';
+    const { getByText, getByAltText } = render(
+      <Quote quote={quoteText} author={authorName} />,
+    );
+
+    const quoteElement = getByText(quoteText);
+    const authorElement = getByText(authorName);
+    const avatarElement = getByAltText(authorName);
+
+    expect(quoteElement).toBeInTheDocument();
+    expect(authorElement).toBeInTheDocument();
+    expect(avatarElement).toBeInTheDocument();
+  });
```

Comment:
- [ ] Some of the tests are failing, Please make sure to fix them and all tests should pass without any errors.

![image](https://github.com/AsumVictor/Math-magician/assets/83803414/f78e5d83-a207-411e-bcb6-bfc6cdb99d79)

Reply:
Thank you very much. we have added more test case and all has ticked passed